example/src/components/buttons/InCallButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactNode } from 'react';
+import { StyleSheet, View, Pressable } from 'react-native';
+
+import { AdditionalColors, BrandColors } from '../../shared/colors';
+
+const IconSize = 25;
+
+const InCallButtonStyles = StyleSheet.create({
+  common: {
+    width: 44,
+    height: 44,
+    borderRadius: 22,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  primary: {
+    borderWidth: 1,
+    borderColor: BrandColors.darkBlue80,
+    borderStyle: 'solid',
+    backgroundColor: AdditionalColors.white,
+  },
+  danger: {
```

Comment:
If it's for disconnect button it's not actually "dangerous". By "dangerous" I've meant buttons like this one:
![Screenshot 2023-03-09 at 15 24 02](https://user-images.githubusercontent.com/6280457/224054122-86f156c2-f7dd-4b94-b24d-ee70d4310e35.png)
Maybe it'd be better to name it "disconnect" or something? Also it's got a frame like a primary button.

Reply:
I didn't want to call it like that, since in the future this might be misunderstood  since as the app get bigger we might want to reuse this styling for something different.

Also I don't think I understand WDYM by "a frame like primary button". I extracted the common part out of their respective stylings.

Reply:
Changed to disconnect.